src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+    { duration: 500, fill: 'forwards' }
+  );
+});
+```
+
+Each time the mouse moves, the browser re-calculates the position for each ball in the comet trail and creates an animation to this new point. The browser now knows to remove old animations (enabling replacement) when:
+
+1. The animation is finished
+2. There is one or more animations higher in composite ordering that are also finished
+3. The new animations are animating the same properties.
+
+You can see exactly how many animations are being replaced by tallying up a counter with each removed animation, using `"anim.onremove"` to trigger the counter.
+
+There are a few additional methods to take your animation control even further:
+
+- `animation.onremove()` calls some custom JavaScript code when an animation is replaced.
+- `animation.replaceState()` provides a means of tracking whether an animation is active, persisted, or removed.
+- `animation.commitStyles()` updates the style of an element based on the underlying style along with all animations on the element in the composite order.
+- `animation.persist()` marks an animation as non-replaceable.
+
+{% Aside %}
+  The last two methods are commonly used with compositing modes, such as "add". Check out the demo below to see them in action.
+{% endAside %}
+
+## Smoother animations with composite modes
+
+With the Web Animations API, you can now set the composite mode of your animations, meaning they can be additive or accumulative, in addition to the default mode of "replace". [Composite modes](https://css-tricks.com/additive-animation-web-animations-api/) allow developers to write distinct animations and have control over how effects are combined. Three composite modes are now supported: `"replace"` (the default mode), `"add"`, and `"accumulate"`.
+
+When you composite animations, a developer can write short, distinct effects and see them combined together. In the example below, we are applying a rotation and scale keyframe to each box, with the only adjustment being the composite mode, added as an option:
+
+<figure class="w-figure">
+<video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/web-animations/replace-add-accumulate.mp4">
+  </video>  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/kevers-google/pen/wvKbaGM"> See Demo on Codepen </a>
+  </figcaption>
+</figure>
+
+In the default, `"replace"` composite mode, the final animation replaces the transform property and ends up at `rotate(360deg) scale(1.4)`. For `"add"`, composite adds the rotation and multiplies the scale, resulting in a final state of `rotate(720deg) scale(1.96)`. `"accumulate"` combines the transformations, resulting in `rotate(720deg) scale(1.8)`. For more on the intricacies of these composite modes, check out the [Web Animations Spec](https://www.w3.org/TR/web-animations-1/#the-compositeoperation-enumeration).
```

Comment:
```suggestion
In the default `'replace'` composite mode, the final animation replaces the transform property and ends up at `rotate(360deg) scale(1.4)`. For `'add'`, composite adds the rotation and multiplies the scale, resulting in a final state of `rotate(720deg) scale(1.96)`. `'accumulate'` combines the transformations, resulting in `rotate(720deg) scale(1.8)`. For more on the intricacies of these composite modes, check out [The CompositeOperation and CompositeOperationOrAuto enumerations](https://www.w3.org/TR/web-animations-1/#the-compositeoperation-enumeration) from the Web Animations spec.
```
* Fixes quotes
* Updates link text to match the name of the section that we're linking to